test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+source settings.sh
+DEVICE=ltsptest01
+
+help() {
+    cat <<-_EOF
+    
+test.sh
+This script creates virtual network adapter which can be used for bridging
+
+Usage:
+$0 [options]
+
+Options:
+    start   This option will create virtual adpater
+    stop    This option will destory virtual adpater
+    
+_EOF
+}
+
+if [ $UID != 0 ]; then
+	echo "error: use sudo or run script as root user"
+	exit
+fi
+
+if [ "$#" == 0 ]; then
+    echo "error: No arguments provided"
+    help
+    exit
+fi
+
+case "$1" in
+    "start" )
+        echo "creating virtual interface.."
+        modprobe dummy
+        ip link add ${DEVICE} type dummy
+        ip link set ${DEVICE} up
+        ip addr add ${NETWORK}.100/24 brd + dev ${DEVICE}
+        echo "INTERFACE=\"${DEVICE}\"" >> settings.sh 
+        ;;
+
+    "stop" )
+        echo "destroying virtual interface.."
+        ip addr del ${NETWORK}.100/24 brd + dev ${DEVICE}
+        ip link delete ${DEVICE} type dummy
+        sed -i settings.sh -e "/^INTERFACE/ c \ " 
+        rmmod dummy
+        ;;
+    * )
+        echo "error: invalid arguments provided"
+        help
+        ;;
```

Comment:
- The default case (`*`) includes also the case when there are no arguments/options.
- Double quotes are not necessary in this case for `$1` and `start`/`stop`. 
- You should study a bit the manual of `sed` (try with `info sed`). The command **c** that you use of the expression stands for `change`. What you need in this case is **d** (for `delete`).